sanic/config.py

```diff
@@ -1,5 +1,11 @@
-class Config:
-    LOGO = """
+import os
+import types
+
+
+class Config(dict):
+    def __init__(self, defaults=None):
+        dict.__init__(self, defaults or {})
```

Comment:
I don't think that built-in dict will inherit a lot of other stuff, but please go with `super` here just in case